ledger-tool/src/blockstore.rs

```diff
@@ -42,13 +42,13 @@ fn analyze_column<
 >(
     db: &Database,
     name: &str,
-) {
+) -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
Return a `BlockstoreError`?

Reply:
On this one, I just went ahead and went straight to returning a `Result<(), LedgerToolError`. Technically, this function can only return a `BlockstoreError` tho ... let me know if you think we should stick with `BlockstoreError` for longer

Reply:
I have a slight preference for using the "lower" error as long as we can (ie. `analyze_column` and `analyze_storage` both return BlockstoreError), but also acknowledge it's not particularly ergonomic to have different Result types in the same module.
I'll leave it up to you. Approving, but happy to re-approve if you're on board with that change.

Reply:
I agree with the sentiment and started making the change. However, I was then reminded that `analyze_column()` (and thus `analyze_storage()`) can return a `serde_json::Error`. Namely, from this line:
```rust
println!("{}", serde_json::to_string_pretty(&json_result)?);
```
So, I can pretend I knew what I was doing when I first implemented this 😉 ; going to push the PR as-is

Reply:
Perfect. Sorry for missing the serde_json error!